### ISSUE TITLE
scene_manager.stats_manager.save_to_csv wrong key word in init file

### ISSUE THREAD

**Issue by oriel-novacy** [Breakthrough/PySceneDetect#299]
In the init file kw argument for the csv file is path but it is an unknown argument in the function:

scene_manager.stats_manager.save_to_csv(path=stats_file_path, base_timecode=video.base_timecode)

it should be - 
scene_manager.stats_manager.save_to_csv(csv_file=stats_file_path, base_timecode=video.base_timecode)

**Comment by oriel-novacy** [comment]
the error is not resolved:
 from running -
 
 ```
scene_list = detect(video_path=video_path,
                    detector=AdaptiveDetector(luma_only=True,min_scene_len=300,window_width=2),
                    show_progress=True)

```
I get the error
```
  File "/home/user/miniconda/lib/python3.8/site-packages/scenedetect/__init__.py", line 193, in detect
    scene_manager.stats_manager.save_to_csv(
TypeError: save_to_csv() got an unexpected keyword argument 'path'
```
that references the function call -
```
scene_manager.stats_manager.save_to_csv(path=stats_file_path, base_timecode=video.base_timecode)
```
I believe the argument "path" should be changed to "csv_file" and it will fix it.
when I change it manually in your code it seems to work fine.

Thanks!!

**Comment by Breakthrough** [comment]
This is fixed in the v0.6.1 branch and will be included in the next release of PySceneDetect.  You can download a development version from [one of the recent builds](https://ci.appveyor.com/project/Breakthrough/pyscenedetect/builds/45440648/job/kvhil1y6d35lq8ka/artifacts) in the meantime. Thanks!
